CHANGELOG.md

```diff
@@ -85,6 +85,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * Switch to auto-generated `String` and `Equal` methods for most proto messages [#1957](https://github.com/provenance-io/provenance/pull/1957).
 * Clean up the marker module's expected BankKeeper interface [#1954](https://github.com/provenance-io/provenance/pull/1954).
 * Add the auto-cli commands and a few others newly added by the SDK [#1971](https://github.com/provenance-io/provenance/pull/1971).
+* Fix unit tests for ibcratelimit [#1977](https://github.com/provenance-io/provenance/pull/1977).
```

Comment:
Can you update `.github/workflows/test.yml` so that the github action runs the now-passing tests again?